services/web/server/src/simcore_service_webserver/projects/_crud_handlers_models.py

```diff
@@ -150,3 +150,19 @@ class Config:
 
 class ProjectActiveParams(BaseModel):
     client_session_id: str
+
+
+class ProjectListFullSearchParams(PageQueryParameters):
+    text: str | None = Field(
+        default=None,
+        description="Multi column full text search, across all folders and workspaces",
+        max_length=100,
+        example="My Project",
+    )
+
+    @validator("text", pre=True)
+    @classmethod
+    def text_check_empty_string(cls, v):
+        if not v:
+            return None
```

Comment:
[DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself): We have this validator in this curated list of common validators: `packages/models-library/src/models_library/utils/common_validators.py` .  Please reuse it